crates/errors/src/error_codes/RT0014.md

```diff
@@ -0,0 +1,7 @@
+## RT0014
+
+The server cannot resume an in-flight invocation which has been started with a now incompatible service protocol version. Restate does not support upgrading service protocols yet.
+
+Suggestions:
+
+* Downgrade the server to a version which is compatible with the used service protocol version
```

Comment:
Another one is "kill the current invocation and restart it"

Reply:
Right now only killing is possible, I guess. Will add it.